crates/ra_hir_def/src/nameres/tests.rs

```diff
@@ -423,31 +423,6 @@ fn extern_crate_rename_2015_edition() {
     );
 }
 
-#[test]
-fn import_across_source_roots() {
-    let map = def_map(
-        "
-        //- /main.rs crate:main deps:test_crate
-        use test_crate::a::b::C;
-
-        //- root /test_crate/
-
-        //- /test_crate/lib.rs crate:test_crate
-        pub mod a {
-            pub mod b {
-                pub struct C;
-            }
-        }
-
-        ",
-    );
-
-    assert_snapshot!(map, @r###"
-        ⋮crate
-        ⋮C: t v
-    "###);
-}
```

Comment:
Why was this removed?

Reply:
That was a single test testing the source root functionality, it seems like it’s worth it to remove the test at the cost of simplifying stuff outside. It’s unlikely that it’ll catch any real bugs